ext/standard/tests/strings/strip_tags_array.phpt

```diff
@@ -0,0 +1,27 @@
+--TEST--
+Test strip_tags() function : basic functionality - with array argument
+--FILE--
+<?php
+
+$string = '<p>foo <b>bar</b> <a href="#">foobar</a></p>';
+var_dump(strip_tags($string));
+var_dump(strip_tags($string, ['a']));
+var_dump(strip_tags($string, ['p', 'a']));
+var_dump(strip_tags($string, []));
+
+var_dump(strip_tags($string, [0]));
+var_dump(strip_tags($string, [1]));
+var_dump(strip_tags($string, [1, 2]));
+var_dump(strip_tags($string, ['color' => 'red', 'item' => 'pen']));
```

Comment:
This test case could use a tag name both as key and value, to show that only the value is relevant.

Reply:
This from strip_tags_variation2.phpt. I added some arrays in the tests with key and values.